infra/main.bicep

```diff
@@ -68,19 +47,33 @@ module postgresServer 'core/database/postgresql/flexibleserver.bicep' = {
       storageSizeGB: 32
     }
     version: '13'
-    administratorLogin: 'flaskadmin'
-    administratorLoginPassword: postgresServerPassword
+    administratorLogin: postgresServerAdmin
+    administratorLoginPassword: postgresAdminPassword
     databaseNames: [postgresDatabaseName]
     allowAzureIPsFirewall: true
   }
 }
 
+module cognitiveService 'core/ai/cognitiveservices.bicep' = {
+  name: 'cognitiveservice'
+  scope: resourceGroup
+  params: {
+    name: cognitiveServiceName
+    location: location
+    tags: union(tags, { 'azd-service-name': 'cognitive' })
```

Comment:
Is { 'azd-service-name': 'cognitive' } needed?

Reply:
Nope, just following the same tagging convention.

Reply:
Ah okay, app service only needs azd-service-name as that matches whats in azure.yaml- its how azd knows where to deploy. its not needed here.

Reply:
Ok, will remove it.